IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DiscreteAccessoryTypeSoundAbsorption/DocPropertySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_DiscreteAccessoryTypeSoundAbsorption" UniqueId="06a176c6-2464-629f-c9ce-291d6151f68b" ApplicableType="IfcDiscreteAccessory/SOUNDABSORPTION" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SoundAbsorptionLimit_3Dyb6v_Lb0uePKjy26Fb4" />
```

Comment:
could we harmonise with maximumNoiseEmissions to have one property?

Reply:
Unit that is used is quite different and the concepts are opposite.
From my side, don't see how to do this..